monai/losses/dice.py

```diff
@@ -105,6 +111,8 @@ def __init__(
         self.smooth_nr = float(smooth_nr)
         self.smooth_dr = float(smooth_dr)
         self.batch = batch
+        self.weight = weight
+        self.register_buffer("class_weight", torch.zeros(1))
```

Comment:
the default is None in the pytorch impl,  does that work? otherwise I think torch.ones is better than torch.zeros as a default value.